datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py

```diff
@@ -1,21 +1,60 @@
 import click
 
 from .....fs import dir_exists, file_exists
-from ...console import CONTEXT_SETTINGS, abort, echo_failure
+from ...console import CONTEXT_SETTINGS, abort, echo_failure, echo_success, echo_warning, echo_info
 from . import validators
 
+from .validators.utils import (
+    initialize_path,
+    exist_profile_in_path
+)
+
 
 @click.command("validate-profile", short_help="Validate SNMP profiles", context_settings=CONTEXT_SETTINGS)
 @click.option('-f', '--file', help="Path to a profile file to validate")
 @click.option('-d', '--directory', help="Path to a directory of profiles to validate")
 @click.option('-v', '--verbose', help="Increase verbosity of error messages", is_flag=True)
-def validate_profile(file, directory, verbose):
-    if file:
-        if not file_exists(file):
-            echo_failure("Profile file not found, or could not be read: " + file)
-            abort()
-    if directory:
-        if not dir_exists(directory):
-            echo_failure("Directory not found, or could not be read: " + directory)
-            abort()
-    validators.validate_profile(file, directory, verbose)
+@click.option('-p', '--path', help="File containg the path of the directories of all profiles")
+def validate_profile(file, directory, verbose, path):
```

Comment:
I'm getting an error like this when running `ddev meta snmp validate-profile`:

```
ddev meta snmp validate-profile
Traceback (most recent call last):
  File ".pyenv/versions/3.8.1/bin/ddev", line 11, in <module>
    load_entry_point('datadog-checks-dev', 'console_scripts', 'ddev')()
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File ".pyenv/versions/3.8.1/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "integrations-core-ws/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py", line 21, in validate_profile
    if not exist_profile_in_path(file,path):
  File "integrations-core-ws/integrations-core/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validators/utils.py", line 55, in exist_profile_in_path
    if isfile(join(directory_path, profile_name)):
  File ".pyenv/versions/3.8.1/lib/python3.8/posixpath.py", line 90, in join
    genericpath._check_arg_types('join', a, *p)
  File ".pyenv/versions/3.8.1/lib/python3.8/genericpath.py", line 152, in _check_arg_types
    raise TypeError(f'{funcname}() argument must be str, bytes, or '
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'NoneType'
```

Reply:
Solved by checking if file was None